src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx

```diff
@@ -0,0 +1,306 @@
+
+#### See your data
+
+In New Relic, you can query your results with this NRQL pattern:
+
+```sql
+FROM activeDirectoryHealthChecks SELECT
+    latest(addressBookClientSessions),
+    latest(inboundFullSyncObjectsRemaining),
+    latest(draInbound),
+    latest(draOutbound),
+    latest(draPendingReplicationSync),
+    latest(directoryReadsPerSec),
+    latest(directoryWritesPerSec),
+    latest(directoryNotifyQueueSize),
+    latest(directoryThreadsInUse),
+    latest(ldapActiveThreads),
+    latest(ldapBindTime),
+    latest(ldapClientSessions),
+    latest(ldapSearchesPerSec),
+    latest(ldapSuccessfulBindsPerSec),
+    latest(contextSwitchesPerSec),
+    latest(processorQueueLength)
+SINCE 1 DAY AGO
+LIMIT MAX
+```
+
+### Windows Services [#windows-services]
+
+Collection of Active Directory Windows Services is accomplished via the native Windows Services integration using this configuration file:
+
+```
+windows-active-directory-services.yml
+```
+
+The results of this collection are stored as dimensional metrics with the decorated label: `label.primary_app = 'active_directory'`.
+
+#### Metric details
+
+|  **SERVICE NAME** |          **DESCRIPTION**         |
+|---------------------|----------------------------------|
+| `ADWS`              | Active Directory Web Services    |
+| `DFS`               | Distributed File System          |
+| `DFSR`              | DFS Replication                  |
+| `DNS`               | DNS Server                       |
+| `Dnscache`          | DNS Client                       |
+| `IsmServ`           | Intersite Messaging              |
+| `kdc`               | Kerberos Key Distribution Center |
+| `lanmanserver`      | Server                           |
+| `lanmanworkstation` | Workstation                      |
+| `Netlogon`          | Net logon                        |
+| `NTDS`              | Active Directory Domain Services |
+| `RpcSs`             | Remote Procedure Call (RPC)      |
+| `SamSs`             | Security Accounts Manager        |
+| `W32Time`           | Windows Time                     |
+
+#### See your data
+
+In New Relic, you can query your results with this NRQL pattern:
+
+```sql
+FROM Metric SELECT
+    latest(timestamp) AS 'Reporting Time',
+    latest(state) AS 'Current State',
+    latest(start_mode) AS 'Start Mode'
+FACET
+    hostname AS 'Host',
+    display_name AS 'Display Name',
+    service_name AS 'Service Name',
+    process_id AS 'Parent PID',
+    run_as AS 'Service Account'
+WHERE label.primary_app = 'active_directory'
+SINCE 1 DAY AGO
+LIMIT MAX
+```
+
+### Active Directory Replication status [#replication-status]
+
+Collection of Active Directory Performance Counters is accomplished via Flex using these two PowerShell scripts and associated configuration file:
```

Comment:
when we say 'Flex' are we referring to our Flex integration? might want to specify and link to it. couple instances in this doc